jet_tracking/devices.py

```diff
@@ -866,3 +910,21 @@ class Diffract(Device, _TableMixin):
                         doc='Diffraction periodogram Frequency analysis amplitude array')
     state = Cpt(EpicsSignal, ':STATE',
                 doc='State of diffraction analysis')
+
+# classes used for jet tracking testing
+class JTInput(Device):
+  nframe = Cpt(EpicsSignal, ':NFRAME', doc='number of frames passed')
+  i0 = Cpt(EpicsSignal, ':i0', doc='Wave8')
+  evtcode = Cpt(EpicsSignal, ':EVTCODE', doc='event code')
+  mtr = Cpt(EpicsSignal, ':MTR', doc='motor position')
+  mtr_prec = Cpt(EpicsSignal, ':MTR_PREC', doc='motor precision')
+
+class JTOutput(Device):
+  nframe = Cpt(EpicsSignal, ':NFRAME', doc='number of frames used')
+  det = Cpt(EpicsSignal, ':DET', doc='detector intensity')
+  i0 = Cpt(EpicsSignal, ':I0', doc='Wave8')
```

Comment:
JTInput i0 PV attribute is ':i0', but output is 'I0'.  I don't know much yet, but is that by convention?

Reply:
I don't see ':i0' or 'I0'. If you mean 'i0' and ':I0' and are talking about the capitalization, then yes, lowercase for variables for PEP8 and uppercase for PVs for some reason.

Reply:
This lives in a gray area since it is referring to a physical quantity, <img src="https://render.githubusercontent.com/render/math?math=I_{0}">, which is unmistakably the upstream intensity.  Admittedly, even the PV suffix kind of sucks because it looks like "IO". 
 Perhaps `i_zero`, `i_naught`, `i_upstream`, or `i_wave8`, or even replacing the "i" with "intensity".  I would also suggest clarifying that `det` also an intensity value -- both in the PV and the python variable.  Perhaps `i_upstream` & `i_downstream` or `i_us` & `i_ds` to keep it generic?

Reply:
I do not have very strong feelings here. Historically, I only remember CAPS in PV names, we have left that behind some time ago. Just as long as we don't have fields starting with an '_'
I prefer zero to naught. Should the PV stay upper case (I_ZERO)? Most PVs are upper case and I personally prefer that, but I won't insist. But if we go lower-case for PVs, let's be consistent within the notebook PVs for jet tracking. I don't like upstream and downstream: in XCS, the liquid scattering ring _is_ the signal and not some measurement downstream of whatever is interesting (which downstream suggests imo).